litestar/types/internal_types.py

```diff
@@ -23,6 +15,9 @@
 
 
 if TYPE_CHECKING:
+    from collections.abc import Callable
+    from typing import Any, Literal
```

Comment:
Nit: since we already import `typing`, we can keep these imports there. This will allow to un-stringify some aliases like `ReservedKwargs`.

But, this is not really important, we can also keep this PR the way it is!

Reply:
Thanks!